tutorial.md

```diff
@@ -90,20 +90,26 @@ export GOPATH=$HOME/go
 export PATH=$PATH:/usr/local/go/bin:$GOPATH/bin
 ```
 
-At this point the Go 1.7.4 tool chain should be installed:
+At this point the Go 1.8.5 tool chain should be installed:
 
 ```
 go version
 ```
 
 ```
-go version go1.7.4 linux/amd64
+go version go1.8.5 linux/amd64
 ```
 
 #### Build crio from source
 
 ```
-sudo apt-get install -y libglib2.0-dev libseccomp-dev libapparmor-dev
+sudo apt-get update && apt-get install -y libglib2.0-dev \
+                                          libseccomp-dev \
+                                          libapparmor-dev \
+                                          libgpgme11-dev \
+                                          libdevmapper-dev \
```

Comment:
Please add skopeo-containers to this list.  You'll get /etc/containrs/policy.json from that.

Reply:
I have an issue trying to install slopes-containers, on ubuntu 16.10. 
As this package is located in ppa:projectatomic/ppa. 
```
Reading package lists... Done
W: The repository 'http://ppa.launchpad.net/projectatomic/ppa/ubuntu yakkety Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: Failed to fetch http://ppa.launchpad.net/projectatomic/ppa/ubuntu/dists/yakkety/main/binary-i386/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.```

Also, when I tried to installed it on 16.04 it's requires dependency ` skopeo : Depends: libostree-1-1 but it is not installable`
https://github.com/kubernetes-incubator/cri-o/issues/817

Reply:
@lsm5 any thoughts on this?

Reply:
Should not need libostree-1.0 if you build with the containers_image_ostree_stub tag.

Reply:
@rhatdan libostree-1.0 it is requires by slopes-containers which creates `/etc/containers/policy.json` which @TomSweeneyRedHat point me to add.

Reply:
skopeo-containers.  I guess if you are installing it, it would be required.

Reply:
Although skopeo-containers should not require libostree, only skopeo if it was built with libostree support.

Reply:
I checked this ubuntu repository,https://launchpad.net/~projectatomic/+archive/ubuntu/ppa?field.series_filter=xenial  and didn't see skopeo-containers package.

Reply:
i'm rebuilding skopeo for ubuntu with libostree removed..

Reply:
@lsm5 please let me know, when you done, and I will update PR.

Reply:
@lsm5 Just double checking.  You're rebuilding skopeo-containers or skopeo?

Reply:
@TomSweeneyRedHat, @lsm5 has updated skopeo package and I added it to this guide. But projectatomic/ppa not working for ubuntu16.10, only for 16.04, @lsm5 will this repository support ubuntu 16.10? Otherwise I need to change OS version.

Reply:
ping @lsm5